build/Build.cs

```diff
@@ -479,9 +479,19 @@ void CompressCalamariProject(Project project)
             _ => _.DependsOn(PackageConsolidatedCalamariZip)
                   .Executes(() =>
                             {
-                                NuGetPack(s => s.SetTargetPath(BuildDirectory / "Calamari.Consolidated.nuspec")
-                                                .SetVersion(NugetVersion.Value)
-                                                .SetOutputDirectory(ArtifactsDirectory));
+                                var releaseNotes = IsLocalBuild ? "Local" : File.ReadAllText(RootDirectory / "releasenotes" / "ReleaseNotes.md");
+                                
+                                var nuspec = BuildDirectory / "Calamari.Consolidated.nuspec";
+                                var text = File.ReadAllText(nuspec);
+                                text = text.Replace("$releaseNotes$", releaseNotes);
```

Comment:
There is a native solution to replace the `$token$` provided by [nuget](https://learn.microsoft.com/en-gb/nuget/reference/nuspec#replacement-tokens). You just need to provide that via a `-properties` argument. This is available as the `.SetProperties()` method inside the `NuGetPack(...)` builder. So you don't need to manually replace the token here.